### PR TITLE
Matt's Bugfix Galore

### LINKED ISSUE #7873: Applying pressure to a bleeding wound causes red blood to cover your hands, even if you are a skrell..
Checklist before you submit an issue! Feel free to partially or fully delete this in your eventual report.
* Do a quick key word search of the git for duplicate reports. If you find any, simply post a reply onto that issue instead!
* Please be specific in your description of the issue. Explain the desired result (what should be happening) and the actual result (what is happening). Simply stating that [something happens] is not very helpful at describing what's wrong.

1.Get shot in the face. It hurts and bleeds so much. You don't have any bandaids, and the doctors cooked their brains on blow so you are SOl for at least 5 minutes.
2. Be a Skrell, a blue-blooded communist. actually this should've been the first step.
3. Clutch your wounds, using help intent, aim for where you are bleeding, click self where it is bleeding: this is your face.
4. Commit a foul divination slash transmutation spell as you have become a red-blooded American, as evidenced by the red blood on your hands.
5. Successfully commit regicide by shooting the last of the hoighty toity tajaran nobles.

* Have you tried reproducing the issue? If yes, steps to reproduce it would help immensely!
I have seen this bug happen twice now.

Edited for Grammer, also Mother Russia Bleeds.

### LINKED ISSUE #8631: MMIs and robot boards can be placed into IPC endoskeletons
As long as the player's ckey inhabiting the MMI or board is whitelisted for IPC, it can be placed into a robot endoskeleton altered to become an unbranded frame, and could presumably be placed into other types of IPCs as well with the right torso from Cargo added. Not only this, but since this is unintended behavior, the player's brain becomes a positronic one once the IPC is activated. This was tested on a private server.

### LINKED ISSUE #9174: Mech wreckages can't be scavenged
Game ID: b7J-b3VG

Mech wreckages are supposed to be cut with welders/plasma cutters and then wrenched into metal. The wreck only reacts to plasma cutters and can't be wrenched into metal.

### LINKED ISSUE #9456: Synthskin Prosthetics Don't Match Skin Tone
Title, basically.
The synthskin prosthesis don't match the skintone of the human that they're attached to, meaning essentially only people who play with the absolute palest skin tone are able to walk around with synthskin prosthesis that match their skin tone. Slightly darker skin tones are viable, if you wear clothing that covers the joints where they're attached, but you can only go so far before the disparity is rather striking. This makes it extremely difficult to play a character with more subtle prosthesis if they aren't both white, and extremely pale at that.

### LINKED ISSUE #9728: Trinary Perfection antenna doesn't render on ear
Simply put, the Trinary Perfection antenna item found in Xenowear - IPC can fit on the ear slot, but does not actually render onmob. It will only render if it is on the helmet slot.

I think it'd be cool if it could render on the ear slot, but if it's unintended, then it's probably prudent to remove it from being able to be placed there.

### LINKED ISSUE #9776: Expedition shotguns have messed up directional sprites when holstered in most of the inventory slots

### LINKED ISSUE #9796: Wristbound Computers can't flip when in the ID Slot
Ideal outcome: flip-wristbound-computer-wrist functioning when the computer is worn on the ID slot.

Present outcome: It does nothing, and only functions when the computer is worn on the gloves slot.

I had tested this a couple times, seeing if taking it off and putting it back on would correct the issue to no avail. This issue was also confirmed by a player by the ckey of Spider4062.

### LINKED ISSUE #9798: AI able to move when unwrenched
Entering and leaving the mech enables the AI to fully function when unwrenched, making the only option to deal with it currently just destroying it. Usually an unwrenched AI can only interact with what it can see, not the entire station. Would be grand if someone could fix that before it gets abused.

### LINKED ISSUE #9799: AI able to use mech while unwrenched
See the other report, unwrenching the core does currently not disable any function of the AI. Please fix that.

### LINKED ISSUE #9807: Borgs cannot pick up docility potions in xenobiology
The purple potion to tripple cores work, the advanced docility potion from light pink slimes does not.

### LINKED ISSUE #9808: One does not simply banish players to the shadow realm
Meme aside, hitting the send to prison button doesnt do anything anymore. The humanity!

### LINKED ISSUE #9809: Ghosts can harvet dionae pods & window to spawn as nymph is far too small
-Ghosts are able to harvest dionae pods. This is fine if:
-The window to play as the dionae is made much larger, like at least 30 seconds.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Fixes #9809 
Fixes #9808 
Fixes #9807 
Fixes #9799 
Fixes #9798 
Fixes #9796 
Fixes #9776 
Fixes blind rune not working
Fixes #9456, synthskin now matches mob skin
Fixes cell runtime on mech explosion
Fixes #9174 wreckage deletion
Fixes #8631 + removes useless virus var on species
Fixes armblade not playing blade sound
Fixes #9728, mobs can now properly target limbs
Fixes #7873

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!bugfix

**Comment by NonQueueingMatt** [comment]
!review

File: code/_onclick/hud/screen_object_types/ai_screen_objs.dm [hunk 1]
```diff
@@ -188,7 +188,10 @@
 /obj/screen/ai/remote_mech/Click()
 	if(isAI(usr))
 		var/mob/living/silicon/ai/AI = usr
-		AI.remote_control_mech()
+		if(AI.anchored)
+			AI.remote_control_mech()
+		else
+			to_chat(AI, SPAN_WARNING("You are unable to get a good connection while unanchored to the station systems."))
```
Comment (Geevies) [inline code/_onclick/hud/screen_object_types/ai_screen_objs.dm:194]:
```suggestion
			to_chat(AI, SPAN_WARNING("You are unable to get a good connection while unanchored from the station systems."))
```

File: code/modules/organs/subtypes/diona.dm [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
-		var/datum/ghosttrap/plant/P = get_ghost_trap("living plant")
-		P.request_player(D, "A diona nymph has split off from its gestalt. ")
-		spawn(60)
-			if(D)
-				if(!D.ckey || !D.client)
-					D.death()
-		return 1
+	INVOKE_ASYNC(src, .proc/create_diona_nymph)
 
+/turf/proc/create_diona_nymph()
```
Comment (Geevies) [inline code/modules/organs/subtypes/diona.dm:9]: please make this a ghostspawner

Comment (Geevies) [reply code/modules/organs/subtypes/diona.dm:9]: you can take a look at my apprentice pebble code to get more info

Comment (NonQueueingMatt) [reply code/modules/organs/subtypes/diona.dm:9]: out of scope

Comment (Geevies) [reply code/modules/organs/subtypes/diona.dm:9]: don't think this was the right PR to do it in, then

**Review by Geevies — APPROVED** [review]

**Pull request merged**